mesa_geo/geoexperimental/UserParam.py

```diff
@@ -0,0 +1,56 @@
+class UserParam:
```

Comment:
This entire file can be removed and to use the `Slider`, you can import from `mesa.experimental`.

Reply:
Yep got it.. in retrospect it seem obvious, ahh well such is life